cmd/cl-adm/config/config.go

```diff
@@ -28,6 +28,10 @@ const (
 	GWCTLInitFile = "gwctl-init.sh"
 	// K8SYamlFile is the filename of the kubernetes deployment yaml file.
 	K8SYamlFile = "k8s.yaml"
+	// K8SSecretYamlFile is the filename of the kubernetes secrets yaml file.
+	K8SSecretYamlFile = "cl-secret.yaml" //nolint:gosec // G101(Potential hardcoded credentials): allow to use secret in the file name.
```

Comment:
Just noticed I should have named the const `K8SYAMLFile`.
Can you please change all `K8SYaml*` to `K8SYAML*`?